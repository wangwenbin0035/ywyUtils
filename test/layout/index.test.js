import layouts from '../../src/layout';

describe('layout', () => {
  test('layout is MatchSnapshot', () => {
    expect(layouts).toMatchSnapshot();
  });
});