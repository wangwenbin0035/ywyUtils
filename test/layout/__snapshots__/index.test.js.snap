// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layout layout is MatchSnapshot 1`] = `undefined`;
